client-app/src/desktop/common/Wrapper.js

```diff
@@ -1,15 +1,63 @@
 import {Component} from 'react';
-import {HoistComponent, elemFactory} from '@xh/hoist/core';
-import {box} from '@xh/hoist/cmp/layout';
+import {HoistComponent, elemFactory, XH} from '@xh/hoist/core';
+import PT from 'prop-types';
+import {box, a, code, div, p} from '@xh/hoist/cmp/layout';
 import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {Icon} from '@xh/hoist/icon';
+import {castArray} from 'lodash';
+import {DockContainerModel, dockContainer} from '@xh/hoist/cmp/dock';
+import {managed} from '@xh/hoist/core/mixins';
 
 import './Wrapper.scss';
 
 @HoistComponent
 class Wrapper extends Component {
+
+    /**
+     * Wrapper is current standard 'wrapper' for demoing hoist-react components. :)
+     * It accepts:
+     *      (optional) `description` for a quick summary of the component,
+     *      (optional) `links` to source code on GitHub or external websites.
+     *
+     * Note that while `links` is optional and that Developers may choose to provide either
+     * a single Object, or array of Objects, each Object requires a `url` and `text` prop:
+     *
+     * @param {string} link.url - can be a full URL (must include https://) or a string that starts with:
```

Comment:
We discussed creating a WrapperLink object, as a place to hang data and still think this is a good idea.

Reply:
I'll start making that